lib/services/api/session.ts

```diff
@@ -1,12 +1,35 @@
 export class Session {
 
-  public token: string
+  public accessToken: string
+  public expireAt?: number
+  public refreshToken?: string
 
   static FromRaw(raw: any) {
-    return new Session(raw.token);
+    return new Session(raw.accessToken, raw.expireAt, raw.refreshToken);
   }
 
-  constructor(token: string) {
-    this.token = token;
+  constructor(accessToken: string, expireAt?: number, refreshToken?: string) {
+    this.accessToken = accessToken;
+    this.expireAt = expireAt;
+    this.refreshToken = refreshToken;
   }
-}
\ No newline at end of file
+
+  private getCurrentTime() {
```

Comment:
This function doesn't appear to be re-used. If it's just use once by `isExpired`, should just be inlined.